src/sage/rings/function_field/drinfeld_modules/drinfeld_module.py

```diff
@@ -1331,7 +1331,7 @@ def is_isomorphic(self, other, absolutely=False):
 
     def is_supersingular(self):
         r"""
-        Return ``True`` whether the Drinfeld module is supersingular.
+        Return ``True`` if the Drinfeld module is supersingular.
```

Comment:
I let @ymusleh decide what's best.

Reply:
As written, 'if' reads better than 'whether' since the latter normally requires more than one possibility. The comment is phrased to say exactly when the code returns true, so there isn't really an option.

I think the "... False otherwise" approach is fine and fairly standard.